coordinates/w64h32.json.example

```diff
@@ -139,7 +139,18 @@
       "append_pitcher_name": false
     },
     "loop": 36,
-    "strikeout": {
+    "play_result": {
+      "x": 15,
+      "y": 29,
+      "font_name": "5x7",
+      "desc_length": "Short"
+    },
+    "hit": {
```

Comment:
These two look like they shouldn't be here?

Reply:
Yes, that's a vestige from before I decided to lump them all under "play_result". It's fixed in the latest commit